sdks/FileSDK.ts

```diff
@@ -70,8 +71,9 @@ export default class FileSDK {
    * with other plugins are resolved it tries to find file from current directory to delete it
    * @returns Promise
    */
-  remove(file: string) {
-    return fse.remove(path.join(this.prefixPath, file));
+  remove(file: string, customPath?: string) {
+    const pathUrl = customPath ? customPath : this.prefixPath;
```

Comment:
This is wrong the constructor should accept the custom path and the instance should use this as prefixPath